src/frontend/src/views/Data/components/SamplesView/components/SampleTableModalManager/components/CreateNSTreeModal/index.tsx

```diff
@@ -99,19 +103,50 @@ export const CreateNSTreeModal = ({
 
   // Filter based on location
   const { data: groupInfo } = useGroupInfo();
-  const { data: namedLocationsData } = useNamedLocations();
-  const namedLocations: NamedGisaidLocation[] =
-    namedLocationsData?.namedLocations ?? [];
+  const { data: namedLocationsData } = useNamedPathogenDepthLocations();
```

Comment:
Now this refetches locations even for covid - I'm not sure where to put the conditional since a hook is not supposed to be inside of one.
@mmmmmmaya do you know off the top of your head?